<ui version="4.0" >
 <class>UserKeyReplaceDialog</class>
 <widget class="QDialog" name="UserKeyReplaceDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Key replacements</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTableWidget" name="keyList" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="gridStyle" >
      <enum>Qt::DashLine</enum>
     </property>
     <property name="rowCount" >
      <number>0</number>
     </property>
     <column>
      <property name="text" >
       <string>Key</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Replacement before word</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Replacement after word</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="addrowButton" >
       <property name="text" >
        <string>add row</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeRowButton" >
       <property name="text" >
        <string>remove row</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
